app/viewmodels/checkAnswers/HumanMedicinesPlasticPackagingWeightSummary.scala

```diff
@@ -24,7 +24,7 @@ import uk.gov.hmrc.govukfrontend.views.viewmodels.summarylist.SummaryListRow
 import viewmodels.govuk.summarylist._
 import viewmodels.implicits._
 
-object HumanMedicinesPlasticPackagingWeightSummary extends SummaryViewModel {
+class HumanMedicinesPlasticPackagingWeightSummary extends SummaryViewModel {
```

Comment:
as discussed, lets turn these back to objects

Reply:
done